src/main/java/jp/co/yahoo/yosegi/spread/column/PrimitiveColumn.java

```diff
@@ -32,7 +32,85 @@
 
 public class PrimitiveColumn implements IColumn {
 
+  public interface ICellMaker {
+    PrimitiveCell create( final PrimitiveObject obj );
+  }
+
```

Comment:
I think this interface class can be replaced to java.util.function and following codes can be written by using lambda functions for simple code.

Reply:
Thank you for your review.
I think that's true.
However, I think coding is simple is a subjective argument.

Would you like to discuss the coding rules separately?

Reply:
Sorry, using lambda function is only a suggestion, and I want to memorize this for after consideration of refactoring or some.

Reply:
It’s difficult the separation of the coding rule and code improve suggestion.
However, I think this kind of argument should be discuss later.
Thank you.